CesiumGltf/include/CesiumGltf/MetadataArrayView.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include "CesiumGltf/PropertyType.h"
+#include <cassert>
+#include <cstddef>
+#include <gsl/span>
+
+namespace CesiumGltf {
+template <typename ElementType> class MetadataArrayView {
+public:
+  MetadataArrayView(gsl::span<const ElementType> buffer)
```

Comment:
`gsl::span<const std::byte>` maybe okay too. My initial thought is the MetadataArrayView is not constructed directly by client, but through PropertyView mostly. Also using `gsl::span<const ElementType>` may save a little storage as we don't need to stores the size of the array. Should I convert it to `gsl::span<const std::byte>`?

Reply:
You could still store a `gsl::span<const ElementType>` internally, but make the constructor take a `gsl::span<const std::byte>` for consistency with the other specializations. And to avoid making clients (even if they're internal) cast. I don't feel strongly about it though so I don't mind if you think it better not to.